src/content/blog/astro-290.mdx

```diff
@@ -0,0 +1,141 @@
+---
+title: "Astro 2.9: View Transitions (experimental)"
+description: "Astro 2.9 is out with experimental support for View Transitions, allowing smooth page transitions."
+publishDate: "July 20, 2023"
+authors:
+  - matthew
+coverImage: ""
+socialImage: ""
+lang: "en"
+---
+
+Astro 2.9 is out and features experiment support for [view transitions](https://developer.chrome.com/docs/web-platform/view-transitions/), as well as redirects configuration and improved script bundling.
+
+- **[View Transitions (experimental)](#view-transitions-experimental)**
+- **[Redirects](#redirects)**
+- **[Hoisted script optimization](#hoisted-script-optimization)**
+
+If you already have Astro installed, you can upgrade it to 2.9 by running the `upgrade` command in your project (using your package manager of choice):
+
+```
+npm install astro@latest
+pnpm upgrade astro@latest
+yarn upgrade astro@latest
+```
+
+While you're at it, upgrade any `@astrojs/*` integrations and adapters you have installed, too!
+
+## View Transitions (experimental)
+
+Astro now supports [view transitions](https://developer.chrome.com/docs/web-platform/view-transitions/) through the new `<ViewTransitions />` component and the `transition:animate` (and associated) directives. View transitions are a great fit for content-oriented sites, and we see it as the best path to get the benefits of client-side routing (smoother transitions) without sacrificing the more simple mental model of MPAs.
```

Comment:
```suggestion
Astro now supports [view transitions](https://developer.chrome.com/docs/web-platform/view-transitions/) through the new `<ViewTransitions />` component and the `transition:animate` directive. Get the benefits of client-side routing (smoother transitions) without sacrificing the simplicity of an MPA.
```
something something, wasn't someone just complaining about everyone using the phrase "mental model" all the time?  :laughing

For the blog post, which doesn't have to be comprehensive docs, let's not "sacrifice the simplicity" of this message!